keyboards/bandominedoni/bandominedoni.h

```diff
@@ -77,9 +77,141 @@
 #define MIS_EIS  LT(_MISC,KC_LNG2)
 #define MIS_KAN  LT(_MISC,KC_LNG1)
 
+//  define which MIDI ch to use.
+//  Note that (actual MIDI ch# - 1) -> 0 .. 15 is used for coding.
+//  ch1
+#define DEFAULT_SUB_CH_NUMBER  0
+//  ch2
+#define ALT_SUB_CH_NUMBER      1
+
+enum custom_keycodes {
+    VERSION = QK_KB,
```

Comment:
You only get 64 keyboard level keycodes <https://github.com/qmk/qmk_firmware/blob/master/data/constants/keycodes/keycodes_0.0.2.hjson#L6>, and can count 75 which would be invalid.

Reply:
Uh, oh…
I can’t use QK_USER in the KB level, can I?

BYW, Thanks for changing the base branch to develop.